v03_pipeline/lib/tasks/write_family_table_test.py

```diff
@@ -522,5 +522,55 @@ def test_gcnv_write_family_table_task(self, mock_dataroot: Mock) -> None:
                         QS=30,
                     ),
                 ],
+                [
+                    hl.Struct(
```

Comment:
Do we prefer to have actual "hl.missing" here, or is a struct with every entry itself `hl.missing` sufficient?  The latter is substantially easier to work with, but may take up more space.

Reply:
either should work for search code, so whatever you prefer

Reply:
Just gonna leave as is.  Easier to reason about.